tests/integration_tests/buffer_test.py

```diff
@@ -722,3 +722,192 @@ def test_capacity_buffer_num():
             count += 1
 
     assert(count == 0)
+
+
+# -------------------------------------------------------------------------- #
```

Comment:
I prefer not having this kind of variable defined at the global scope. Ideally, this template would be in a setup function (see http://doc.pytest.org/en/latest/fixture.html ) or if it's only used in one test, put it in the test function itself.